modules/zarr/src/lib/storage.ts

```diff
@@ -0,0 +1,51 @@
+import { fetchFile } from '@loaders.gl/core';
+import { KeyError } from 'zarr';
+import type { AsyncStore } from 'zarr/types/storage/types';
+
+/**
+ * Preserves (double) slashes earlier in the path, so this works better
+ * for URLs. From https://stackoverflow.com/a/46427607/4178400
```

Comment:
```suggestion
 * for URLs. From https://stackoverflow.com/a/46427607
```

FYI you don't need the last part of the SO url. It's just your user id, and IMO the shorter the URL the better for code comments.